### PR TITLE
Benchmark comparison

### PR DESCRIPTION
**Pull request by paigerube14**
Using common scripts to get elastic search resutls and be able to compare for ingress and k8s-netperf 

Ingress perf comparison: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/250/console

k8s-netperf: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/234/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -145,12 +150,26 @@ pipeline {
...
+            branches: [[name: "main_scripts" ]],
+            doGenerateSubmoduleConfigurations: false,
+            extensions: [
+                [$class: 'CloneOption', noTags: true, reference: '', shallow: true],
+                [$class: 'PruneStaleBranch'],
+                [$class: 'CleanCheckout'],
+                [$class: 'IgnoreNotifyCommit'],
+                [$class: 'RelativeTargetDirectory', relativeTargetDir: 'helpful_scripts']
+            ],
+            userRemoteConfigs: [[url: "https://github.com/paigerube14/ocp-qe-perfscale-ci.git" ]]
```
Comment (qiliRedHat) [inline Jenkinsfile:171]: @paigerube14 Did you intend to use your fork?

**Comment by qiliRedHat** [comment]
@SachinNinganure Could you review the network-perf? You can get the .scv file form Build Artifacts.

**Comment by qiliRedHat** [comment]
@paigerube14 Thanks so much for making this happen! For ingress-perf, it looks good to me. Now it compares the total_avg_rps, I hope to add latencies comparisons as well. This can be done in the future. We can firstly monitor the rps compre for a while.

**Comment by qiliRedHat** [comment]
BTW, a minor issue I noticed, no need to be changed in this pr.
The ingress-perf comparison job [https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/250/console](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/250/console,), the grafana link for ingress-perf is incorrect: 
https://grafana.rdu2.scalelab.redhat.com:3000/d/8wDGrVY4k/kube-burner-compare-update?orgId=1&var-Datasource=QE%20kube-burner&var-sdn=OVNKubernetes&var-workload=ingress-perf&var-worker_nodes=&var-latencyPercentile=P99&var-condition=Ready&var-component=kube-apiserver&var-uuid=15faf7fd-d137-4ee8-b8f0-be1fc9244614&var-uuid=aa3c6182-f70a-42d0-bfd8-ed5c3e7cf183

I use the ingress-perf dashboard to do ingress-perf comparison, a sample link:
https://grafana.rdu2.scalelab.redhat.com:3000/d/nlAhmRyVk/ingress-perf?orgId=1&var-datasource=QE%20Ingress-perf&var-platform=AWS&var-clusterType=self-managed&var-uuid=c9b08c17-6a10-4c64-8ad6-8c4db90b843c&var-uuid=15fcccb7-2f27-46c5-86af-22edd1bcc0b6&var-termination=edge&var-termination=http&var-termination=passthrough&var-termination=reencrypt&var-latency_metric=avg_lat_us&var-compare_by=uuid.keyword&from=now-24h&to=now

I feel it's not easy to handle the from&to because the BASELINE_UUID can be anytime.

File: Jenkinsfile (hunk 1 above)
Comment (paigerube14) [reply Jenkinsfile:171]: I have to have my fork set for now as the code hasn't been merged, if you can merge [this pr](https://github.com/openshift-qe/ocp-qe-perfscale-ci/pull/535) I will update to main repo/branch

Just don't want to break if people are using/testing it out

**Comment by paigerube14** [comment]
> BTW, a minor issue I noticed, no need to be changed in this pr. The ingress-perf comparison job [https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/250/console](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/250/console,), the grafana link for ingress-perf is incorrect:  https://grafana.rdu2.scalelab.redhat.com:3000/d/8wDGrVY4k/kube-burner-compare-update?orgId=1&var-Datasource=QE%20kube-burner&var-sdn=OVNKubernetes&var-workload=ingress-perf&var-worker_nodes=&var-latencyPercentile=P99&var-condition=Ready&var-component=kube-apiserver&var-uuid=15faf7fd-d137-4ee8-b8f0-be1fc9244614&var-uuid=aa3c6182-f70a-42d0-bfd8-ed5c3e7cf183
> 
> I use the ingress-perf dashboard to do ingress-perf comparison, a sample link: https://grafana.rdu2.scalelab.redhat.com:3000/d/nlAhmRyVk/ingress-perf?orgId=1&var-datasource=QE%20Ingress-perf&var-platform=AWS&var-clusterType=self-managed&var-uuid=c9b08c17-6a10-4c64-8ad6-8c4db90b843c&var-uuid=15fcccb7-2f27-46c5-86af-22edd1bcc0b6&var-termination=edge&var-termination=http&var-termination=passthrough&var-termination=reencrypt&var-latency_metric=avg_lat_us&var-compare_by=uuid.keyword&from=now-24h&to=now
> 
> I feel it's not easy to handle the from&to because the BASELINE_UUID can be anytime.

+ 1 I had just had it set for kube-burner jobs, will update for ingress and network. If possible I would like this to not block the pr if everything else looks good

**Comment by paigerube14** [comment]
@qiliRedHat updated ingress perf link. Couldn't find a good grafana dashboard yet for network-perf-v2

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/benchmark-comparison/254/

**Comment by qiliRedHat** [comment]
@paigerube14 looks good to me, would you merge the commits?

**Comment by SachinNinganure** [comment]
I have run network tests for this with some config changes as this code works for kube-burner jobs only, Paige acked me the same as I was hitting some errs. I will be able to verify network tests once there changes added.
Ty

**Comment by paigerube14** [comment]
@qiliRedHat commits are squashed, ptal

**Pull request merged**